content/blog/tyranny-of-nits/index.md

```diff
@@ -0,0 +1,208 @@
++++
+title = "The tyranny of nits"
+description = "How open source work sputters and dies"
+date = 2023-09-23
+author = "Alice I. Cecile"
+
+[taxonomies]
+tags = ["open source", "development philosophy"]
++++
+
+Open source software development runs on code review.
+It's a great way to transfer knowledge about what's going on and best practices,
+to align on goals and vision,
+and perhaps most importantly, it's one of the only tools to avoid [complete chaos](https://helixpedia.fandom.com/wiki/Twitch_Plays_Pokemon) when literally anyone could change your code.
+
+Accordingly, if your code review process sucks, helping out your open source project is going to suck.
+There's a lot of different ways it could suck, to be fair: you could be [drowning in triage and backlog hell](https://www.leafwing-studios.com/blog/triage-by-controversy/), or you could simply have a [terrible person or three](https://geekfeminism.fandom.com/wiki/Missing_stair) making everything miserable.
+
+But this post isn't about that.
+It's about the talented and principled engineers who through their well-intentioned suggestions and feedback grind contributions to a halt.
+
+You see, the limiting factor in open source isn't time, and it isn't effort.
+It's "give-a-fuck".
```

Comment:
I don't love this wording, because it's more like "willingness to put in effort". I'd probably rewrite this to talk about how even the most determined or enthusiastic contributor can be worn down by endless, pointless bullshit. Not a huge deal though.

Reply:
I want to be more biting here. I also disagree with it being about "effort": there's a huge element of good will that isn't really captured by that framing.

Reply:
Alright, I'm convinced